src/network/networkd-lldp-tx.c

```diff
@@ -184,6 +195,34 @@ static int lldp_make_packet(
                 p = mempcpy(p, pretty_hostname, pretty_hostname_length);
         }
 
+        if (mud) {
+                uint8_t oui_mud[sizeof(SD_LLDP_OUI_MUD)] = {0x00, 0x00, 0x5E};
```

Comment:
why initialize this you memcpy() SD_LLDP_OUI_MUD over it anyway

Reply:
ohh . Sry missed. Dropped now thanks.